park_api/cities/Regensburg.py

```diff
@@ -0,0 +1,54 @@
+from bs4 import BeautifulSoup
+from park_api.util import convert_date
+from park_api.geodata import GeoData
+import datetime
+
+# This loads the geodata for this city if <city>.geojson exists in the same directory as this file.
+# No need to remove this if there's no geodata (yet), everything will still work.
+geodata = GeoData(__file__)
+
+# This function is called by the scraper and given the data of the page specified as source in geojson above.
+# It's supposed to return a dictionary containing everything the current spec expects. Tests will fail if it doesn't ;)
+def parse_html(html):
+
+    # BeautifulSoup is a great and easy way to parse the html and find the bits and pieces we're looking for.
+    soup = BeautifulSoup(html, "html.parser")
+
+    # last_updated is the date when the data on the page was last updated, it should be listed on most pages
+    #   suche: <p class="updateinfo">zuletzt aktualisiert: 28.05.2019 15.30 Uhr</p>
+    updated = soup.find( "p", class_="updateinfo")
+    last_updated = datetime.datetime.strptime(updated.text, 'zuletzt aktualisiert: %d.%m.%Y %H.%M Uhr')
```

Comment:
Why do you use `datetime` at all here? The `convert_date` function does is exactly that. So the simply (and correct) solution would be `last_updated = convert_date(updated.text, 'zuletzt aktualisiert: %d.%m.%Y %H.%M Uhr')` There's no need for you to use `datetime` by yourself. A further problem (aside from avoidable code duplication) is that your code doesn't handle time zones correctly so the timestamps would be wrong here.

If you take a look at https://github.com/offenesdresden/ParkAPI/blob/48b35e0a5590412664207eda4f48c355e7b040ce/park_api/util.py#L83-L98 `convert_date` you'll see that we do exactly the same already but with correct timezone handling. Please replace this code with `convert_date`.

Reply:
when I started with the Wiesbaden-project I was new to Python. One of the problems I encountered was using the convert_date-function because the import didn't work. So I used the datetime-function and from that moment on I continued using this function.

And the import still does not work here in my Eclipse-environment. So I have to find a solution for this.

Reply:
I don't know how eclipse handles this. We usually use a virtual environment. If you enable the environment, install the package there and then start eclipse from within that environment (on the command line) it should pick up the correct python paths which then should also make the packages available. This depends on eclipse picking up the system configuration. If eclipse has its own handling about packages etc. I don't know how to fix that, but I don't think that this is the case.
Example:
```
$ virtualenv -p python3 venv  #create virtual environment, needs only to be done once
$ . venv/bin/activate         #activate environment, needs to be done in each shell
$ pip install -e .            #install dependencies, needs only to be done once, or after updates
$ eclipse &
```